--- config.def.h	2019-02-02 20:55:02.000000000 +0800
+++ config.h	2019-07-22 22:22:19.343082967 +0800
@@ -6,11 +6,13 @@
 static const char *fonts[] = {
 	"monospace:size=10"
 };
+
 static const char *prompt      = NULL;      /* -p  option; prompt to the left of input field */
+/* Has Gruvbox light theme */
 static const char *colors[SchemeLast][2] = {
-	/*     fg         bg       */
-	[SchemeNorm] = { "#bbbbbb", "#222222" },
-	[SchemeSel] = { "#eeeeee", "#005577" },
+	           /*     fg         bg       */
+	[SchemeNorm] = { "#ffeeff", "#282828" },
+	[SchemeSel] = { "#1d2021", "#a89984" },
 	[SchemeOut] = { "#000000", "#00ffff" },
 };
 /* -l option; if nonzero, dmenu uses vertical list with given number of lines */
